pkg/plugins/resource_fungibility/resource_fungibility.go

```diff
@@ -151,7 +155,16 @@ func (rf *ResourceFungibility) calPreFilterState(ctx context.Context, pod *v1.Po
 		return nil
 	}
 
-	for _, f := range model.Spec.InferenceConfig.Flavors {
+	// Filter the flavors from model by the inference service flavors annotation.
+	selectedFlavors := model.Spec.InferenceConfig.Flavors
```

Comment:
What do it means? Can you provide more description on this?

Reply:
added.